Makefile

```diff
@@ -66,6 +66,7 @@ help:
 	@echo "    unicharset       Create unicharset"
 	@echo "    lists            Create lists of lstmf filenames for training and eval"
 	@echo "    training         Start training"
+	@echo "    traineddata      Create .traineddata files from each checkpoint
```

Comment:
A closing `"` was missing here. I fixed that and improved the build rule a little.

Reply:
Great, thanks! It's my first time dealing with makefiles, feel free to point out any mistake